src/test/scala/org/apache/pekko/persistence/dynamodb/snapshot/SnapshotTooBigSpec.scala

```diff
@@ -0,0 +1,62 @@
+/*
```

Comment:
new files should use the standard ASF header like the one in https://github.com/apache/incubator-pekko/blob/main/project/AddMetaInfLicenseFiles.scala

only if the code is largely copied from Akka should it have the special 'derived from Akka' header